### PR TITLE
[Bug][P1] Fix location argument parsing error

### PR DESCRIPTION
**Pull request by deerhunter4**
README.md file updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +24,15 @@ meteostat.net,5cc4a3480mshd9...p111424jsn9e365e801bf8
...
 Example of basic usage:  
-`python main.py -location <city>`
+`python3 main.py -location <city>`
 
 Example of optional arguments usage:  
-`python main.py -location “New York” -forecast_period 72h -date 12-06-2024`
+`python3 main.py -location “New York” -forecast_period 72h -date 12-06-2024`
+
+**Usage (Windows)**  
+Running a program in Windows powershell with the above code might give an error. In that case check if Python3 is installed, by typing `python --version`. If yes, then run the above code again changing 'python3' to 'python'.
```
Comment (KrzysiekJodlowski) [inline README.md]: @deerhunter4 this does not seem to be fully right, are we 100% sure it's a pure Linux vs Windows issue and not the local configuration one? Also, what about other systems like MacOS?

I would suggest being more cautious on this point and changing this text to:
```
**Usage:**  
Example of basic usage:  
`python3 main.py -location <city>`

Example of optional arguments usage:  
`python3 main.py -location “New York” -forecast_period 72h -date 12-06-2024`

> **_NOTE:_**
The Better_Weather app and the libraries it uses require Python 3, as indicated in the examples. It is worth noting that on Windows, if Python 3 is installed via an executable installer, the Python executable might be set to `python` by default, especially if no other versions of Python are installed. For more detailed information on setting up Python and managing different versions, please refer to the [official Python documentation](https://docs.python.org/3/using/index.html).
```

**Review by KrzysiekJodlowski — CHANGES_REQUESTED** [review]
@deerhunter4 please consider the suggested changes. Also, again I see that *This branch has conflicts that must be resolved* - please fix that for this branch/PR as well.

File: README.md (hunk 1 above)
Comment (deerhunter4) [reply README.md]: @KrzysiekJodlowski 
I agree, no it is more precise. Thanks.

Comment (KrzysiekJodlowski) [reply README.md]: @deerhunter4 thanks for addressing that; please also remove the `(Linux)` part from [here](https://github.com/deerhunter4/Better_Weather/blob/f937aaa9cb7e1cbfa7bc70d2ea1f2151e3ff35ae/README.md?plain=1#L37) as I mentioned above.

Comment (deerhunter4) [reply README.md]: @KrzysiekJodlowski sorry, an oversight. Now should be ok.

**Review by KrzysiekJodlowski — APPROVED** [review]

**Pull request merged**